app/manifest/v2/brave.json

```diff
@@ -1 +1,7 @@
-{}
+{
```

Comment:
Was there anything about webpack that made this change necessary? Or is it just a nice to have?

Reply:
Yes, brave needs the same CSP as the chrome build. This file isn't used by the gulp build system (it only uses chrome and firefox).

gulp build documentation only permits using the manifest files for chrome and firefox, per `development/build/index.js`, though the runtime itself doesn't directly disallow it:

```
.option('platform', {
          default: '',
          description:
            'Specify a single browser platform to build for. Either `chrome` or `firefox`',
          hidden: true,
          type: 'string',
        })
        ```
        
        
An alternative is to delete these (and update some spots in the webpack build to remove any references to browsers we don't build for directly).

Reply:
:heavy_check_mark: